oid_lookup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import requests
+
+def get_oid_title(oid):
```

Comment:
Nice Job digging through some sloppy hacks of mine.

We might think about the run-time dependency this puts on a 3rd party site, as well as possible fire-wall issues.

Reply:
I have another idea about OID only snooping, but worthy of a f2f convo and scoping. Likely not worth the cycles at this point in time